src/Client.cpp

```diff
@@ -199,13 +199,12 @@ void CClient::ReadLine(const CString& sData) {
 		return;                       // Don't forward this msg.  We don't want the client getting us disconnected.
 	} else if (sCommand.Equals("PROTOCTL")) {
 		VCString vsTokens;
-		VCString::const_iterator it;
 		sLine.Token(1, true).Split(" ", vsTokens, false);
 
-		for (it = vsTokens.begin(); it != vsTokens.end(); ++it) {
-			if (*it == "NAMESX") {
+		for (auto it : vsTokens) {
```

Comment:
This makes a copy of the string. Please default to `const auto&`.
If it needs to be changed, and not in copy, `auto&` will do.